### PR TITLE
Add the command to push helm repo

### PR DESCRIPTION
**Pull request by knanao**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -53,6 +53,18 @@ else
 	helm package manifests/$(MOD) --version $(VERSION) --app-version $(VERSION) --dependency-update --destination .artifacts
 endif
 
+.PHONY: push
```
Comment (khanhtc1202) [inline Makefile:56]: should add a note about push command to Line 10

File: Makefile [hunk 2]
```diff
@@ -53,6 +54,18 @@ else
...
+push/chart: BUCKET ?= charts.pipecd.dev
+push/chart: VERSION ?= $(shell git describe --tags --always --dirty --abbrev=7)
+push/chart: CREDENTIALS_FILE ?= ~/.config/gcloud/application_default_credentials.json
+push/chart:
+	@yq '.version = "${VERSION}"' -i manifests/pipecd/Chart.yaml
+	@yq '.version = "${VERSION}"' -i manifests/piped/Chart.yaml
+	@yq '.version = "${VERSION}"' -i manifests/site/Chart.yaml
+	@yq '.version = "${VERSION}"' -i manifests/helloworld/Chart.yaml
+	docker run --rm -it -v ${CREDENTIALS_FILE}:/secret -v ${PWD}:/repo gcr.io/pipecd/chart-releaser:1.2.0 /chart-releaser --bucket=${BUCKET} --manifests-dir=repo/manifests --credentials-file=secret
```
Comment (khanhtc1202) [inline Makefile:66]:
```suggestion
	docker run --rm -it -v ${CREDENTIALS_FILE}:/secret -v ${PWD}:/repo gcr.io/pipecd/chart-releaser:1.2.0 /chart-releaser --bucket=${BUCKET} --manifests-dir=repo/manifests --credentials-file=${CREDENTIALS_FILE}
```
Should it be like this?

[the remark this answers is not in the document]
Comment (knanao) [reply Makefile:66]: ok, let's fix it.

Comment (knanao) [reply Makefile]: Nope, this is mounted to `/secret`, so this is fine.

File: Makefile [hunk 3]
```diff
@@ -53,6 +54,18 @@ else
...
+.PHONY: push
+push/chart: BUCKET ?= charts.pipecd.dev
+push/chart: VERSION ?= $(shell git describe --tags --always --dirty --abbrev=7)
+push/chart: CREDENTIALS_FILE ?= ~/.config/gcloud/application_default_credentials.json
+push/chart:
+	@yq '.version = "${VERSION}"' -i manifests/pipecd/Chart.yaml
+	@yq '.version = "${VERSION}"' -i manifests/piped/Chart.yaml
+	@yq '.version = "${VERSION}"' -i manifests/site/Chart.yaml
+	@yq '.version = "${VERSION}"' -i manifests/helloworld/Chart.yaml
+	docker run --rm -it -v ${CREDENTIALS_FILE}:/secret -v ${PWD}:/repo gcr.io/pipecd/chart-releaser@sha256:fc432431b411a81d7658355c27ebaa924afe190962ab11d46f5a6cdff0833cc3 /chart-releaser --bucket=${BUCKET} --manifests-dir=repo/manifests --credentials-file=secret #v0.13.0
```
Comment (khanhtc1202) [inline Makefile:66]:
```suggestion
	docker run --rm -it -v ${CREDENTIALS_FILE}:/secret -v ${PWD}:/repo gcr.io/pipecd/chart-releaser@sha256:fc432431b411a81d7658355c27ebaa924afe190962ab11d46f5a6cdff0833cc3 /chart-releaser --bucket=${BUCKET} --manifests-dir=repo/manifests --credentials-file=${CREDENTIALS_FILE} #v0.13.0
```

**Pull request merged**